plugins/GSdx/GSLinuxDialog.cpp

```diff
@@ -500,8 +500,9 @@ bool RunLinuxDialog()
 	GtkWidget* osd_box      = gtk_vbox_new(false, 5);
 
 	// Grab a logo, to make things look nice.
-	GdkPixbuf* logo_pixmap = gsdx_ogl_logo;
-	GtkWidget* logo_image  = gtk_image_new_from_pixbuf(logo_pixmap);
+	GResource * resources = GSdx_res_get_resource();
+	GdkPixbuf * ogl_logo = gdk_pixbuf_new_from_stream(g_resource_open_stream(resources,"/GSdx/res/logo-ogl.bmp",G_RESOURCE_LOOKUP_FLAGS_NONE,NULL),NULL,NULL);
```

Comment:
There is a potential memory leak.

https://developer.gnome.org/gio/stable/GResource.html#g-resource-open-stream
```
Returns

GInputStream or NULL on error. Free the returned object with g_object_unref(). 
```